docs/UserGuide.adoc

```diff
@@ -236,25 +252,40 @@ show you information such as:
 
 * `NAME`: student name
 * `MATRIC_NUMBER`: matriculation number
-* `RATING`: ratings you have given the student (on a scale from 1 - 5)
+* `RATING`: ratings you have given the student, represented by stars (on a scale from 1 - 5)
```

Comment:
Can rating only be whole numbers? If so, we should mention it. Otherwise ppl may try to put decimals.